version.bzl

```diff
@@ -0,0 +1 @@
+VERSION = '3.17.3'
```

Comment:
Also, are there any runbooks/instructions we need to update to ensure this is updated properly when the current release process is running? @acozzette might have some ideas here.

Reply:
I figured out how to do this and will create another PR to edit the update version script.